modules/client/examples/03-multisig-client/06-can-execute.ts

```diff
@@ -0,0 +1,33 @@
+/* MARKDOWN
+### Check if user can execute an action in a multisig plugin
+
+Checks whether the signer of the transaction is able to execute actions created by proposals from the Multisig plugin.
+*/
+import {
+  CanExecuteParams,
+  ContextPlugin,
+  MultisigClient
+} from "@aragon/sdk-client";
+import { context } from "../00-setup/00-getting-started";
+
+// Instantiate a plugin context from the aragonOSx SDK context.
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Instantiate a Multisig client.
+const multisigClient: MultisigClient = new MultisigClient(contextPlugin);
+
+const canExecuteParams: CanExecuteParams = {
+  pluginAddress: "0x1234567890123456789012345678901234567890", // the address of the plugin contract itself.
+  proposalId: BigInt(0)
+};
+
+// Checks whether the signer of the transaction can execute a given proposal.
```

Comment:
This feels inaccurate. Approved proposals can be approved by anyone, being a signer or not

Reply:
Is this taken from the source code? If so, we also need to fix it there

Reply:
Fixed!